core/server/worker/src/main/java/alluxio/worker/block/SpaceReserver.java

```diff
@@ -31,22 +31,26 @@
 import javax.annotation.concurrent.NotThreadSafe;
 
 /**
- * {@link SpaceReserver} periodically checks if there is enough space reserved on each storage tier,
- * if there is no enough free space on some tier, free space from it.
+ * The {@link SpaceReserver} periodically checks the available space on each storage tier. If the
+ * available space is above the high watermark configured for the tier, eviction will be
```

Comment:
does "available space" being "above" mean there is more space than a threshold? Don't we want eviction to happen when the available space is "below a threshold"?

Reply:
Changed the wording, thanks.